examples/terraform/aws/variables.tf

```diff
@@ -173,47 +173,47 @@ variable "ami_filters" {
     ubuntu = {
       owners       = ["099720109477"] # Canonical
       image_name   = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
-      osp_name     = "osp-ubuntu"
+      osp_name     = ""
```

Comment:
Let's remove this field completely. We already have the variable `initial_machinedeployment_operating_system_profile` that users can use to specify a custom OSP i.e. if they don't want the defaulting from MC webhooks.

Reply:
good point, deleted